VideoLocker/src/main/java/org/edx/mobile/view/CourseUnitVideoFragment.java

```diff
@@ -158,16 +161,38 @@ public void onActivityCreated(Bundle savedInstanceState) {
             isLandscape = false;
         } else {
             isLandscape = true;
```

Comment:
You can remove this if else. Nothing uses `isLandscape`. You can also remove variable declaration.